src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.header-logo {
+  height: 40px; /* cuz a have height 43,3px  */
```

Comment:
Haven't figured out what does this comment mean

Reply:
It's answer of your future question: "why is it here?"))